src/Components/Endpoints/src/DependencyInjection/RazorComponentsServiceCollectionExtensions.cs

```diff
@@ -27,9 +27,10 @@ public static class RazorComponentsServiceCollectionExtensions
     /// Registers services required for server-side rendering of Razor Components.
     /// </summary>
     /// <param name="services">The service collection.</param>
+    /// <param name="configure">An <see cref="Action{RazorComponentOptions}"/> to configure the provided <see cref="RazorComponentsOptions"/>.</param>
     /// <returns>An <see cref="IRazorComponentsBuilder"/> that can be used to further configure the Razor component services.</returns>
-    [RequiresUnreferencedCode("Razor Components does not currently support trimming or native AOT.", Url = "https://aka.ms/aspnet/trimming")]
-    public static IRazorComponentsBuilder AddRazorComponents(this IServiceCollection services)
+    [RequiresUnreferencedCode("Razor Components does not currently support trimming or native AOT.", Url = "https://aka.ms/aspnet/nativeaot")]
```

Comment:
Why was this URL changed? I believe the intention in https://github.com/dotnet/aspnetcore/pull/49377 was for this URL to be https://aka.ms/aspnet/trimming.